api/v1alpha1/rollout_types.go

```diff
@@ -33,6 +33,8 @@ type RolloutSpec struct {
 	ObjectRef ObjectRef `json:"objectRef"`
 	// rollout strategy
 	Strategy RolloutStrategy `json:"strategy"`
+	// RolloutID should be an uuid that distinguishes each rollout progress.
```

Comment:
RolloutID is not necessary an uuid, any user defined identifying string can be used as rollout id